staff/sergio-luz/app-template/home.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>home</title>
+</head>
+<body>
+    <header>
+        <h1>Skylab Coders Academy</h1>
+    </header>
+    <nav class="topnav" id="myTopnav">
+        <a href="home.html" class="active">Home</a>
```

Comment:
group all links in a list (ul > li). do not leave links just like that, unlisted.